cvat/apps/dataset_manager/tests/test_formats.py

```diff
@@ -496,6 +511,21 @@ def test_frames_outside_are_not_generated(self):
             self.assertTrue(frame.frame in range(6, 10))
         self.assertEqual(i + 1, 4)
 
+    def test_can_import_annotations_for_image_with_dots_in_filename(self):
+        export_formats = [f.DISPLAY_NAME for f in dm.views.get_export_formats()]
+        formats = [f.DISPLAY_NAME for f in dm.views.get_import_formats() if
```

Comment:
This definition will miss few formats, check this declaration https://github.com/openvinotoolkit/cvat/blob/develop/cvat/apps/engine/tests/test_rest_api.py#L4414